app/src/main/res/values/strings.xml

```diff
@@ -187,9 +197,38 @@
         <item></item>
     </string-array>
 
+    <string-array name="ads1115_gain_spinner">
+        <item>GAIN_TWOTHIRDS</item>
+        <item>GAIN_ONE</item>
+        <item>GAIN_TWO</item>
+        <item>GAIN_FOUR</item>
+        <item>GAIN_EIGHT</item>
+        <item>GAIN_SIXTEEN</item>
+    </string-array>
+
+    <string-array name="ads1115_channel_spinner">
+        <item>UNI_0</item>
+        <item>UNI_1</item>
+        <item>UNI_2</item>
+        <item>UNI_3</item>
+        <item>DIFF_01</item>
+        <item>DIFF_23</item>
+    </string-array>
+
+    <string-array name="ads1115_rate_spinner">
+        <item>8</item>
+        <item>16</item>
+        <item>32</item>
+        <item>64</item>
+        <item>128</item>
+        <item>250</item>
+        <item>475</item>
+        <item>860</item>
+
```

Comment:
end string array by tag ```</string-array>```. And Please check the build once before pushing the commit.